materialize-databricks/sqlgen.go

```diff
@@ -198,23 +220,9 @@ SELECT -1, ""
 {{ end -}}
 {{ end }}
 
-{{ define "cast" }}
-{{- if Contains $ "DATE" -}}
-	::DATE
-{{- else if Contains $ "TIMESTAMP" -}}
-	::TIMESTAMP
-{{- else if Contains $ "DOUBLE" -}}
-	::DOUBLE
-{{- else if Contains $ "BIGINT" -}}
-	::BIGINT
-{{- else if Contains $ "BOOLEAN" -}}
-	::BOOLEAN
-{{- else if Contains $ "BINARY" -}}
-	::BINARY
-{{- else if Contains $ "STRING" -}}
-	::STRING
-{{- end -}}
-{{ end }}
+{{ define "cast" -}}
+::{{- First (Split $ " ") -}}
```

Comment:
This may not work with every possible custom DDL (e.g. `INTERVAL` or `MAP` or `ARRAY` might fail to work with this cast), but I didn't want to invest more time in this for now

Reply:
Perhaps just a TODO comment then?